src/openassetio-core/include/openassetio/hostApi/ManagerFactory.hpp

```diff
@@ -29,7 +30,10 @@ OPENASSETIO_DECLARE_PTR(ManagerFactory)
  */
 class OPENASSETIO_CORE_EXPORT ManagerFactory final {
  public:
-  using Identifiers = std::vector<Str>;
+  /// Unique identifier of a manager.
+  using Identifier = Str;
```

Comment:
[consider] As this is common to both interfaces, and the wrappers, for both host and manager, would lit make sense to add this to `typedefs.hpp` or similar?

Reply:
Added a global `Identifier` alias and used throughout the codebase in 88df07b1

Reply:
[question] Are we happy to keep using `StringView` in C for these?

Reply:
Well grepped. Swapped more places to use `Identifier` in 59aa6e86

I think `StringView` is fine for now, I kinda see `Identifier` as more of a documentation thing. If we did want to swap `Identifier` for some other type, then chances are it would still need to be convertible to a `char` array for C API purposes.